packages/dapi/scripts/tx-filter-stream.js

```diff
@@ -143,10 +153,33 @@ async function main() {
     wrapInErrorHandler(subscribeToTransactionsWithProofsHandler),
   );
 
+  // eslint-disable-next-line operator-linebreak
+  const subscribeToBlockHeadersWithChainLocksHandler =
+    subscribeToBlockHeadersWithChainLocksHandlerFactory(
+      getHistoricalBlockHeadersIterator,
+      dashCoreRpcClient,
+      dashCoreZmqClient,
+      subscribeToNewBlockHeaders,
+    );
+
+  const wrappedSubscribeToBlockHeadersWithChainLocks = jsonToProtobufHandlerWrapper(
+    jsonToProtobufFactory(
+      BlockHeadersWithChainLocksRequest,
+      PBJSBlockHeadersWithChainLocksRequest,
+    ),
+    protobufToJsonFactory(
+      PBJSBlockHeadersWithChainLocksResponse,
+    ),
+    wrapInErrorHandler(subscribeToBlockHeadersWithChainLocksHandler),
+  );
+
   const grpcServer = createServer(
     getCoreDefinition(0),
     {
       subscribeToTransactionsWithProofs: wrappedSubscribeToTransactionsWithProofs,
+      // TODO: should we rename tx-filter-stream.js to something different,
+      // having in mind that it now holds subscribeToBlockHeadersWithChainLocks?
```

Comment:
Make sense. I would use something like `core-streams`

Reply:
Let's do in a separate chore PR, so that it is visible in CHANGELOG?